aws/resource_aws_s3_bucket.go

```diff
@@ -396,6 +400,20 @@ func resourceAwsS3Bucket() *schema.Resource {
 													Type:     schema.TypeString,
 													Optional: true,
 												},
+												"access_control_translation": {
+													Type:     schema.TypeSet,
+													Optional: true,
+													MinItems: 1,
+													MaxItems: 1,
+													Elem: &schema.Resource{
```

Comment:
Need to add `Set: accessControlTranslationHash,` so that the correct set element hash function is used.

Reply:
This attribute does not need to be `TypeSet` since it is more of a "container" attribute. This will be updated on merge.